js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/invertedIndex/useInvertedIndexJSONSchema.tsx

```diff
@@ -12,7 +14,8 @@ const invertedIndexJSONSchema: JSONSchemaType<InvertedIndexValuesType> = {
     },
     name: {
       nullable: true,
-      type: "string"
+      type: "string",
+      pattern: extendedNames ? "" : "^[a-zA-Z][a-zA-Z0-9-_]*$"
```

Comment:
nit: inside character classes I prefer `-` literals to be escaped for better readability.
```suggestion
      pattern: extendedNames ? "" : "^[a-zA-Z][a-zA-Z0-9\-_]*$"
```